### DISCUSSION TITLE
Combine pseudo-classes into one

### DISCUSSION THREAD

**Discussion by AndreyGunderin** [acss-io/atomizer#511] [Q&A] [answered]
Good day, community.

Is it possible right now to combine pseudo-classes like this: `C(red):h:f` ?
Or we can use now only separate classes like this: `C(red):h C(red):f` ?

Regards, Andrew.

**Comment by redonkulus** [comment, marked as answer]
Hi, thanks for the question. You will have to declare the classes separately. We do not support multiple pseudo-class suffixes on one class.
